spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -572,6 +600,17 @@ def save_figure_as(self, fig, fmt):
         if fname:
             save_figure_tofile(fig, fmt, fname)
 
+    def copy_current_figure(self):
+        """Copy the currently selected figure."""
+        if self.current_thumbnail is None:
+            return
+
+        b_succeed = copy_figure(self.current_thumbnail.canvas.fig,
+                                self.current_thumbnail.canvas.fmt)
+
+        if b_succeed:
+            QMessageBox.information(self, "Information", "copy completed")
```

Comment:
First, this should be ``Plot copied to clipboard`` so as to be clearer and look more polished.

Second, in my testing, the message does appear when clicking the copy button in the menu, and when pressing ``Ctrl-C`` with the focus on the the main plot window or a plot in the sidebar, but *not* when selecting the Copy item in either the main plot window context menu or the sidebar one. It doesn't make sense to show it some places but not others, so we should be consistent if we are going to—either on all three, on the keyboard shortcut only, or none of them.

Third, I'm not sure its good UX to show it at all; the command and KB shortcut are already exposed in the toobar and context menus. Further, its only really useful the first time using the KB shortcut when the user might not know they can copy the plot there, and after that it just gets in the way and is an extra modal dialog that gets in the way and blocks the user from doing anything else any time they copy something.

Reply:
I agree with your opinion.
There is no animation when click button on the toolbar. So I added messagebox. I will consider another way to notify that the copy button is pressed.

Reply:
@CAM-Gerlach 
I add feature to blink image when click copy button. How do you think about it?

![blink](https://user-images.githubusercontent.com/22970578/50373185-57014f80-061e-11e9-99a7-a297ffd87b19.gif)

And I inserted the following code to blink the image of thumbnail.

    def set_frame_size(self, width, height):
        self.setFixedSize(width, height)

    def blink_image(self):
        timer = QTimer()
        frame_rect = self.frameSize()
        self.setFixedSize(0, 0)
        timer.singleShot(100,
                         lambda: self.set_frame_size(frame_rect.width(),
                                                     frame_rect.height()))

Reply:
> There is no animation when click button on the toolbar.

This is a problem with the dark theme, then, and should be fixed (there is one in the light theme). I'll report it over there.

> I add feature to blink image when click copy button. 

Good thinking! I like that solution, personally. If we're going to apply it to the toolbar (where there will be a button clicked indication once that gets fixed) we may as well apply it to all methods of copying an image, including the context menu, to be consistent.